test/lib/redis-store-spec.js

```diff
@@ -257,6 +257,30 @@ describe('del', function () {
     });
   });
 
+  it('should delete multiple values for a given array of keys', function (done) {
+    redisCache.set('foo', 'bar', function () {
+      redisCache.set('bar', 'baz', function () {
+        redisCache.set('baz', 'foo', function () {
+          redisCache.del(['foo', 'bar', 'baz'], function (err) {
+            assert.equal(err, null);
+            done();
```

Comment:
I can add that, if you'd like, but I ran the tests with the monitor command and got:
```
1499930687.577381 [0 172.19.0.1:50500] "setex" "foo" "60" "\"bar\""
1499930687.578115 [0 172.19.0.1:50500] "setex" "bar" "60" "\"baz\""
1499930687.578880 [0 172.19.0.1:50500] "setex" "baz" "60" "\"foo\""
1499930687.580916 [0 172.19.0.1:50500] "multi"
1499930687.580935 [0 172.19.0.1:50500] "del" "foo"
1499930687.580941 [0 172.19.0.1:50500] "del" "bar"
1499930687.580944 [0 172.19.0.1:50500] "del" "baz"
1499930687.580948 [0 172.19.0.1:50500] "exec"
1499930687.582961 [0 172.19.0.1:50500] "setex" "foo" "60" "\"bar\""
1499930687.583707 [0 172.19.0.1:50500] "setex" "bar" "60" "\"baz\""
1499930687.584318 [0 172.19.0.1:50500] "setex" "baz" "60" "\"foo\""
1499930687.585478 [0 172.19.0.1:50500] "multi"
1499930687.585495 [0 172.19.0.1:50500] "del" "foo"
1499930687.585501 [0 172.19.0.1:50500] "del" "bar"
1499930687.585505 [0 172.19.0.1:50500] "del" "baz"
1499930687.585508 [0 172.19.0.1:50500] "exec"
```
So, I'm confident they are being deleted.